### ISSUE TITLE
project rating workn't

### ISSUE THREAD

**Issue by KumarGanesha1996** [Ultimate-Dogecoin-Infinity-2/ethereum-auction-platform#3]
hi kind sirs

i rate it project 2 stars but readme not update.

see:: 
<img width="195" alt="image" src="https://user-images.githubusercontent.com/65182868/134745298-792b0ab0-b67e-40d7-835c-af66fe1bd849.png">

still 5 stars, but i rate 2.

please fix !

regards,
Kumar Ganesha

**Comment by KumarGanesha1996** [comment]
@marcin-serwin @Wowol @krzysztofpioro please take action asap !

**Comment by KumarGanesha1996** [comment]
hi kind sirs,

i do you're job for you pls acept. - https://github.com/Ultimate-Dogecoin-Infinity-2/ethereum-auction-platform/pull/4

regards,
Kumar Ganesha

**Comment by marcin-serwin** [comment]
Our project rating is based on an extremely complicated evaluation aggregation algorithm called `maximum` (or `max` as we like to call it). You can read more about it on the [wikipedia page](https://en.wikipedia.org/wiki/Maxima_and_minima), though it is higher level mathematics so consider yourself warned. If you manage to understand the process of computation you will clearly see that the current rating didn't change despite a new rating made by you. This is precisely why we have chosen the function as it doesn't require us to update the README as frequently as other common options such as `mean` or  `median`.

In conclusion thank you for the issue and the proposed pull request but for the reasons outline above I must close it.

**Comment by KumarGanesha1996** [comment]
thank sir for explanation !

regards,
Kumar
